scripts/autogenHelpers/DocJSONBuilder.js

```diff
@@ -26,24 +26,24 @@ const IGNORE_FILES = [
 
 const IGNORE_METHODS = ['setNativeProps'];
 
+const fileExtensionsRegex = /.(js|tsx|(?<!d.)ts)$/;
+
 class DocJSONBuilder {
   constructor(styledLayers) {
     this._styledLayers = {};
 
     for (const styleLayer of styledLayers) {
-      const ComponentName = pascelCase(styleLayer.name);
-      this._styledLayers[
-        ComponentName +
-          (['Light', 'Atmosphere', 'Terrain'].includes(ComponentName)
-            ? ''
-            : 'Layer')
-      ] = styleLayer;
+      let ComponentName = pascelCase(styleLayer.name);
+      const fakeLayers = ['Light', 'Atmosphere', 'Terrain'];
+      if (!fakeLayers.includes(ComponentName)) {
+        this._styledLayers[ComponentName + 'Layer'] = styleLayer;
+      }
```

Comment:
```suggestion
      } else {
         this._styledLayers[ComponentName + 'Layer'] = styleLayer;
      }
```

It's missing compared to original code, that's why Atmosphere is not generating.

Reply:
🤦 yep.